sklearn/model_selection/_search.py

```diff
@@ -555,6 +559,22 @@ def classes_(self):
         self._check_is_fitted("classes_")
         return self.best_estimator_.classes_
 
+    def _generate_candidates(self):
+        """Generates lists of candidates to search as a coroutine
```

Comment:
-> "Generate lists of candidate parameters to evaluate."?

Having our conversation in mind I still had to think twice to grok this, so I'll try and make a suggestion for the doc string that would have helped me.